.mergify.yml

```diff
@@ -0,0 +1,17 @@
+# See https://doc.mergify.io
+
+pull_request_rules:
+- name: auto_merge
+  conditions: 
+    - -title~=(WIP|wip)
```

Comment:
is this robust enough or would we improve on it over time somehow?

Reply:
There will be a changeset introduced in a later PR to expand on this